apps/spruce/src/pages/distroSettings/tabs/SingleTaskDistroTab/getFormSchema.ts

```diff
@@ -0,0 +1,45 @@
+import { GetFormSchema } from "components/SpruceForm";
+
+export const getFormSchema = (): ReturnType<GetFormSchema> => ({
+  fields: {},
+  schema: {
+    type: "object" as const,
+    properties: {
+      projectTasksPairs: {
+        type: "array" as const,
+        title: "Allowed Tasks",
+        items: {
+          type: "object" as const,
+          properties: {
+            displayTitle: {
+              type: "string" as const,
+              title: "Project ID / Repo",
+            },
+            allowedTasks: {
+              type: "array" as const,
+              title: "",
+              items: {
+                title: "Task Regex",
+                type: "string" as const,
+              },
+            },
+          },
+        },
+      },
+    },
+  },
+  uiSchema: {
+    projectTasksPairs: {
+      "ui:readonly": true,
+      "ui:orderable": false,
+      "ui:useExpandableCard": true,
+      "ui:description":
```

Comment:
Should we add any disclaimers about this not being available to all users yet? Since this PR will launch the page.

Reply:
I think this is okay for now since users would have to file a ticket to add their tasks anyways.